client/src/stores/NetworkStore.js

```diff
@@ -20,14 +20,18 @@ var NetworkStore = Reflux.createStore({
     });
     this._updateNetwork(null);
   },
+  onSetIpfs: function(ipfsInstance, callback) {
+    this.orbit._ipfs = ipfsInstance;
```

Comment:
The IPFS instance is passed to orbit in the constructor, like `orbit = new Orbit(ipfs)`, so this will break Orbit. We need to do it the other way around: first start the daemon, when started, create Orbit instance (now in App in componentDidMount()) and then follow the rest of the steps in App.componentDidMount() to start the app.

If you do the refactoring to IpfsDaemonStore as proposed above, then you can do all this in one place in App.onDaemonStarted().